### PR TITLE
Fix string when creating a new offer

### PR DESCRIPTION
**Pull request by BtcContributor**
Now it is similar to the message provided when taking a new offer.

See: https://github.com/bisq-network/bisq/blob/3047cd48d42c50f908f5750a8dccea194c057c69/core/src/main/resources/i18n/displayStrings.properties#L541

### CODE REVIEW

**Comment by ripcurlx** [comment]
@m52go Could you please ACK this change? Thanks!

File: core/src/main/resources/i18n/displayStrings.properties [hunk 1]
```diff
@@ -471,7 +471,7 @@ createOffer.placeOfferButton=Review: Place offer to {0} bitcoin
 createOffer.createOfferFundWalletInfo.headline=Fund your offer
 # suppress inspection "TrailingSpacesInProperty"
 createOffer.createOfferFundWalletInfo.tradeAmount=- Trade amount: {0} \n
-createOffer.createOfferFundWalletInfo.msg=You need to deposit {0} to this offer.\n\n\
+createOffer.createOfferFundWalletInfo.msg=You need to deposit {0} for making this offer.\n\n\
```
Comment (m52go) [inline core/src/main/resources/i18n/displayStrings.properties]:
```suggestion
createOffer.createOfferFundWalletInfo.msg=You need to deposit {0} to make this offer.\n\n\
```

**Comment by BtcContributor** [comment]
@m52go so should we also change this https://github.com/bisq-network/bisq/blob/3047cd48d42c50f908f5750a8dccea194c057c69/core/src/main/resources/i18n/displayStrings.properties#L541 ?

Like that: You need to deposit {0} to take this offer

**Comment by m52go** [comment]
> @m52go so should we also change this 

Yes, that would be better.

**Comment by m52go** [comment]
ACK

**Review by ripcurlx — APPROVED** [review]
utACK

**Pull request merged**